tabpy-server/tabpy_server/management/util.py

```diff
@@ -14,23 +12,25 @@
 from datetime import datetime, timedelta, tzinfo
 from time import mktime
 
-from tabpy_server.common.tabpy_logging import (
-    PYLogging, log_error, log_info, log_debug, log_warning)
-import logging
+
 logger = logging.getLogger(__name__)
-PYLogging.initialize(logger)
+
 
 def load_state_from_config_file(ps_state):
     '''
     Fill initial TabPy state from state file path
     '''
     try:
-        log_info("Loading state from state file")
+        logger.info("Loading state from state file")
         state_file_path = os.environ['TABPY_STATE_PATH']
+
+        # TODO: the below is namespaced with util which isn't valid (?)
+        # check if this is dead code and remove accordingly
         config = util._get_state_from_file(state_file_path)
```

Comment:
It is stated in the comment above but unless I am overlooking something this code has to always error out since `util` is not a valid namespace in this context. This is probably dead code that can be removed altogether but that review and execution is suitable for a separate change